examples/azure/terraform-azure-cosmosdb-example/main.tf

```diff
@@ -0,0 +1,88 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY AN AZURE VIRTUAL MACHINE
+# This is an example of how to deploy an Azure Virtual Machine with the minimum network resources.
+# ---------------------------------------------------------------------------------------------------------------------
+# See test/azure/terraform_azure_example_test.go for how to write automated tests for this code.
+# ---------------------------------------------------------------------------------------------------------------------
+
+provider "azurerm" {
+  version = "~> 2.29"
+  features {}
+}
+
+
+# ---------------------------------------------------------------------------------------------------------------------
+# PIN TERRAFORM VERSION TO >= 0.12
+# The examples have been upgraded to 0.12 syntax
+# ---------------------------------------------------------------------------------------------------------------------
+
+terraform {
+  # This module is now only being tested with Terraform 0.13.x. However, to make upgrading easier, we are setting
+  # 0.12.26 as the minimum version, as that version added support for required_providers with source URLs, making it
+  # forwards compatible with 0.13.x code.
+  required_version = ">= 0.12.26"
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY A RESOURCE GROUP
+# ---------------------------------------------------------------------------------------------------------------------
+
+resource "azurerm_resource_group" "rg" {
+  name     = var.resource_group_name
+  location = var.location
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY A COSMOSDB ACCOUNT
+# ---------------------------------------------------------------------------------------------------------------------
+
+resource "azurerm_cosmosdb_account" "test" {
+  name                = var.cosmosdb_account_name
+  location            = azurerm_resource_group.rg.location
+  resource_group_name = azurerm_resource_group.rg.name
+  offer_type          = "Standard"
```

Comment:
we want to use the smallest instance/sku sizes to reduce cost during CI as it runs on every pull request multiple times usually.  If this is smallest, that is fine.

Reply:
Yes, the cost of Cosmos DB is based on RU/s (request units per second), the minimum being 400.